test/unit/test_logger.py

```diff
@@ -126,3 +126,28 @@ def test_cyan_text():
     x = '{}{}{}'.format(colorama.Fore.CYAN, 'foo', colorama.Style.RESET_ALL)
 
     assert x == logger.cyan_text('foo')
+
+
+def test_markup_detection_pycolors_0(monkeypatch):
```

Comment:
I think this and the next test could be collapsed into one parametrized test case